dotnetv4/Bedrock-runtime/Actions/HelloBedrockRuntime.cs

```diff
@@ -28,7 +28,8 @@ private static async Task Invoke(string modelId, string prompt)
                 default:
                     Console.WriteLine($"Unknown model ID: {modelId}. Valid model IDs are: {CLAUDE}.");
                     break;
-            };
+            }
```

Comment:
this was the result of running `dotnet format` in order to make sure linter passes